app/models/developer.rb

```diff
@@ -24,6 +24,7 @@ class Developer < ApplicationRecord
   has_one :referring_user, through: :user
   has_many :conversations, -> { visible }
   has_many :messages, -> { where(sender_type: Developer.name) }, through: :conversations
+  has_many :specialty, through: :specialty_tags
```

Comment:
I think this should be pluralized. You may also need to supply a `source`. I'm a bit rusty on the exact association definition here.
```
has_many :specialties, through: :specialty_tags, source: :specialty
```

Reply:
I'm not sure why this is needed at all. Up at the top of this file we have `HasSpecialties` which contains `has_many :specialties, -> { visible }, through: :specialty_tags, dependent: :destroy`.

Can you please remove this line and update the query to use the existing relationship?